frontend/src/pages/about/TermsOfService/sections/ChangesToTheseTerms.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { Alert, AlertTitle, Box, Link, Typography } from '@mui/material';
+
+import { githubTournesolTermsOfServiceHistoryUrl } from 'src/utils/url';
+
+const ChangesToTheseTerms = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Box>
+      <Typography
+        variant="h4"
+        fontStyle="italic"
+        gutterBottom
+        id="changes-to-these-terms"
+      >
+        {t('terms.changesToTheseTerms.changesToTheseTerms')}
+      </Typography>
+      <Box my={2}>
+        <Alert severity="info">
+          <AlertTitle>
+            <strong>{t('terms.shortVersion')}</strong>
+          </AlertTitle>
+          {t('terms.changesToTheseTerms.shortVersion')}
+        </Alert>
+      </Box>
+      <Typography paragraph>
+        <Trans t={t} i18nKey="terms.changesToTheseTerms.weMayAmendTheseTerms">
+          We may amend these Terms of Service at any time. We will notify the
+          Users of material changes to this Agreement, at least 30 days prior to
+          the change taking effect by posting a notice on our website or sending
+          email to the email address specified in your Account. User&apos;s
+          continued use of the Service after those 30 days constitutes agreement
+          to those revisions of this Agreement. For any other non-impacting
+          modifications, User&apos;s continued use of the Service constitutes
+          agreement to our revisions of these Terms of Service. You can view all
+          changes to these Terms in our{' '}
+          <Link
+            href={githubTournesolTermsOfServiceHistoryUrl}
```

Comment:
This URL to a jsx file is not helpful for anyone, right? I recommend removing it completely.

Or change the link to a page showing the list of commits with a tag "[terms]" in them and forcing ourselves to not forget adding this tag in commit messages when we make these changes.

Reply:
I just realized that you are right, it's not useful because I've split the Terms of Service into several files.

> Or change the link to a page showing the list of commits with a tag "[terms]" in them and forcing ourselves to not forget adding this tag in commit messages when we make these changes.

It's a good idea, even if we can still make mistakes by writing term singular instead of plural, or by simply omitting the tag. This can be mitigated by creating a PR template or by documenting our nomenclature in the CONTRIBUTING.md.

If we follow our currently non-documented nomenclature we should use in our commit messages:

```bash
# the first part is the software affected, the second part is the scope
[front] terms: add a minimal age requirement 

# or shorter, but not explicit
[front] tos: add a minimal age requirement 
```

It will also make the GitHub search more accurate because searching for "[terms]" only will return all commits with the word "terms", even if they are not related to the terms of service (the symbols `[` and `]` are omitted). On the other hand searching for "[front] terms" will allow us to search explicitly for commits starting with "[front] terms" which have a greater chance to match our expectations : )

We can use the GitHub search as an history with the appropriate search query. Here is an example with "[front] style".

https://github.com/search?q=repo%3Atournesol-app%2Ftournesol+type%3Acommit+sort%3Acommitter-date-desc+%22%5Bfront%5D+style%22&type=commits

We can manually run and test this query by using the search field with:

```
repo:tournesol-app/tournesol type:commit sort:committer-date-desc "[front] style"
```